test/integration_test.py

```diff
@@ -271,7 +274,7 @@ def _list_partition_bundles(self,
         fqids = self.azul_client.list_bundles(catalog, source, partition_prefix)
         bundle_count = len(fqids)
         partition = f'Partition {effective_prefix!r} of source {source.spec}'
-        if not config.is_sandbox_or_personal_deployment:
+        if True:
```

Comment:
I don't think we want this.

Reply:
Sorry, it's necessary to test on `nadove5`, which does not use a common prefix in order to replicate the config for `anvilprod`. Committing the change was an oversight.